docs/ops.html

```diff
@@ -1218,11 +1218,7 @@ <h4 class="anchor-heading"><a id="multitenancy-more" class="anchor-link"></a><a
   <h3 class="anchor-heading"><a id="java" class="anchor-link"></a><a href="#java">6.5 Java Version</a></h3>
 
   Java 11, Java 17, Java 21 and Java 23 are supported.
-  <p>
-  Note that Java 11 support for the broker and tools has been deprecated since Apache Kafka 3.7. Both will be removed in Apache Kafka 4.0.
-  <p>
-  Java 11 and later versions perform significantly better if TLS is enabled, so they are highly recommended (they also include a number of other
-  performance improvements: G1GC, CRC32C, Compact Strings, Thread-Local Handshakes and more).
+
```

Comment:
Seems we should still clarify that Java 11 is not supported for brokers any longer, but only for clients/connect/ks ?

Reply:
Sounds good, I can do that.